### ISSUE TITLE
code error

### ISSUE THREAD

**Issue by liao1123** [avitalsh/jamming_attack#1]
There's a bug in the initialization code where the loss sign is inconsistent with the optimization process.
attack.py 
226 lines
`
cur_loss = dist_res2res_target[-1]
loss_hist = [cur_loss]
`
but
300 lines
` 
batch_dist_res2res_target = np.mean(np.abs(score_func_oracle(res_emb_oracle, res_target_emb_oracle)), axis=1)
loss = - batch_dist_res2res_target
best_cand_idx = np.argmin(loss)
`
